apps/transport/lib/jobs/geojson_converter_job.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Transport.GeojsonConverterJob do
+  @moduledoc """
+  Job converting a GTFS file to GeoJSON
+  """
+  use Oban.Worker, max_attempts: 1
+  import Logger
+  alias DB.{Repo, Resource}
+
+  # TO DO: handle the case where a resource cannot be found
+
+  @impl true
+  def perform(%{id: id, args: %{"resource_id" => resource_id}}) do
+    Logger.info("Job #{id} started by #{__MODULE__}")
+
+    url = Resource |> Repo.get!(resource_id) |> Map.fetch!(:url)
+
+    # TO DO how is the tmp folder cleaned (upon completion or after a crash)?
+    gtfs_file_path = System.tmp_dir!() |> Path.join("#{id}_download")
+
+    # TO DO stream file to disk
+    # TO DO verify headers (content-type) and maybe provide alerts to providers!
+    %{status: 200, body: body} = Unlock.HTTP.Client.impl().get!(url, [])
```

Comment:
Pourquoi le client HTTP Unlock et pas celui de Transport ?

Reply:
Celui-ci utilise [Finch](https://hexdocs.pm/finch/Finch.html) qui est plus performant. Faudrait sûrement qu'on fasse le ménage la dedans. Je vais ouvrir une issue.

Reply:
@fchabouis si tu ouvres un ticket, on pourra mentionner dedans https://github.com/wojtekmach/req, qui est une surcouche au dessus de Finch, et supporte les redirections etc (et d'autres avantages).

Reply:
@fchabouis a noté, je clôture